### PR TITLE
fix(DB/Creature): Dreadsaber & Shardhorn Rhino positioning

### PR DESCRIPTION
**Pull request by aleigood**
Fix creatures come together and don't move randomly.

76700890-f9f82a80-66f6-11ea-8cdb-8a8652db527e

## TESTS PERFORMED:
Has tested on linux, works fine.

## HOW TO TEST THE CHANGES:
.go xyz 5600.802734 5510.528809 -90.812248 571

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
Can you explain which issue you fixed?
Maybe with screenshots?

Also you can use https://www.deepl.com/translator for language translation

**Comment by aleigood** [comment]
> Can you explain which issue you fixed with this Pull Request?
> Maybe with screenshots?
> 
> Also you can use https://www.deepl.com/translator for language translation

capture uploaded.

**Comment by Kitzunu** [comment]
ERROR 1062 (23000) at line 77575: Duplicate entry '1585757753946835500' for key 'PRIMARY'

This error is caused by
```sql
INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1585757753946835500');
```

File: data/sql/updates/pending_db_world/rev_1585757753946835500.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1585757753946835500');
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1585757753946835500.sql:1]: @aleigood Fix SQL rev.
http://www.azerothcore.org/wiki/How-to-create-a-PR

Thanks @Kitzunu

**Comment by Si1ker** [comment]
@aleigood Thanks.

@Kitzunu
1- Is the SQL rev working for you now?
2- What about the creatures? Good?

**Comment by Kitzunu** [comment]
I havent tested the PR but you still have travis error.
Still due to the rev key.
https://travis-ci.org/github/azerothcore/azerothcore-wotlk/jobs/670422190#L624

**Comment by aleigood** [comment]
I don't know where the problem is, I used the file generated by createsql.sh according to the instructions.

**Comment by Kitzunu** [comment]
I would believe it to be that you have not updated your Database with the latest updates from the repository and therefore it is generating an "old" key.

Am I right in that assumption @FrancescoBorzi @BarbzYHOOL @Helias ?

**Comment by Helias** [comment]
@Kitzunu, yes, should be, try to update your database and generate again the pending sql version

**Comment by Kitzunu** [comment]
Now all builds are passing :) Good job @BarbzYHOOL

**Comment by Kitzunu** [comment]
Ty for the PR!

**Pull request merged**